File: app/src/main/java/com/satish/nytimesmvvm/pojo/Article.java
package com.satish.nytimesmvvm.pojo;

import androidx.room.Entity;
import androidx.room.PrimaryKey;
import com.fasterxml.jackson.annotation.JsonIgnore;
import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
import com.fasterxml.jackson.annotation.JsonProperty;

@Entity(tableName = "article_table")
@JsonIgnoreProperties(ignoreUnknown = true)
public class Article {
    @JsonIgnore
    @PrimaryKey(autoGenerate = true)
    private int id = 0;
    @JsonProperty("title")
    private String title ;
    @JsonProperty("abstract")
    private String articleAbstract;
    @JsonProperty("thumbnail_standard")
    private String thumbnailUrl;

    public int getId() {
        return id;
    }

    public void setId(int id) {
        this.id = id;
    }

    public String getTitle() {
        return title;
    }

    public void setTitle(String title) {
        this.title = title;
    }

    public String getArticleAbstract() {
        return articleAbstract;
    }

    public void setArticleAbstract(String articleAbstract) {
        this.articleAbstract = articleAbstract;
    }

    public String getThumbnailUrl() {
        return thumbnailUrl;
    }

    public void setThumbnailUrl(String thumbnailUrl) {
        this.thumbnailUrl = thumbnailUrl;
    }
}

File: app/src/main/java/com/satish/nytimesmvvm/pojo/Article.kt
package com.satish.nytimesmvvm.pojo

import androidx.room.Entity
import androidx.room.PrimaryKey
import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName

@Entity(tableName = "article_table")
data class Article (
    @PrimaryKey(autoGenerate = true)
    var id: Int = 0,
    @Expose
    @SerializedName("title")
    var title: String = "",
    @Expose
    @SerializedName("abstract")
    var abstract: String = "",
    @Expose
    @SerializedName("thumbnail_standard")
    var thumbnailUrl: String = ""
)